### ISSUE TITLE
Bad decoding to chinese characters

### ISSUE THREAD

**Issue by Eagles2F** [nylas/N1#4]
![screen shot 2015-10-05 at 10 30 56 am](https://cloud.githubusercontent.com/assets/3294244/10288512/2c73bc8a-6b4e-11e5-866d-22fdfdb2c656.png)

**Comment by dillonforrest** [comment]
@Eagles2F thank you for downloading N1 and for sharing this issue with us! We really appreciate it. :)

**Comment by emorikawa** [comment]
@Eagles2F Yes, I was able to reproduce this. Thank you so much for reporting. It appears to be an issue with the backend sync-engine and Gmail with certain unicode sets. I have filed a bug with our internal tracking system as well as a corresponding ticket (https://github.com/nylas/sync-engine/issues/212) on the [nylas/sync-engine](https://github.com/nylas/sync-engine) repo. I'm going to close this ticket here in favor of tracking it on the sync-engine.
